### ISSUE TITLE
SequelSequencers

### ISSUE THREAD

**Issue by danieldavies99** [VCVRack/community#703]
Hi, I would like to submit my plugin Sequel Sequencers:
Source: https://github.com/danieldavies99/sequel
Branch: master

Thanks!

**Comment by cschol** [comment]
Crash on module load: https://github.com/danieldavies99/sequel/issues/4

**Comment by danieldavies99** [comment]
> Crash on module load: https://github.com/danieldavies99/sequel/issues/4

Thanks, I'll look into this and get back to you

**Comment by danieldavies99** [comment]
@cschol This should be fixed by [#5](https://github.com/danieldavies99/sequel/pull/5) which is now merged into master.

**Comment by cschol** [comment]
Your plugin has been integrated and will be available for all supported platforms with the next Plugin Manager update. Please let us know in **this** issue when a new version is available. We will reopen it. Thank you for providing your plugin to the Rack community.

**Comment by danieldavies99** [comment]
@cschol I have made an updated version of this module following some user feedback, as such I would like to reopen this issue, 

version 2.0.1 commit: https://github.com/danieldavies99/sequel/commit/ddc455925000f0768bb811b73e839c953966b14c

**Comment by danieldavies99** [comment]
@cschol @AndrewBelt Hi, I have added a new module (Sequel 16) and I have done some general code cleanup/refactoring:

V2.1.0 commit:
https://github.com/danieldavies99/sequel/commit/12dddd08f037c0e3ab71a5c7f955938f45a6f42f

Please could you reopen this issue, thanks!

**Comment by danieldavies99** [comment]
@cschol @AndrewBelt Hi, I would like to deprecate both modules in this plugin, in favour of the Daniel Davies plugin I have just submitted. As such I would like to reopen this issue:

deprecation commit: https://github.com/danieldavies99/sequel/commit/b2f6fde517f722c971423f15da7ff0ff3e28606d

**Comment by cschol** [comment]
@danieldavies99 Here is what we recommend: add your new plugins to the `SequelSequencers` plugin. Do not create a **new** plugin, like `DanielDavies`. Version 2.1.1 of Rack will add a compatibility override for `SequelSequencers` plugin slug. So, anyone who used the `SequelSequencers` plugin will automatically updated to a new plugin name. After release, you update the plugin slug of `SequelSequencers` to `DanielDavies`. Does that make sense? @AndrewBelt anything I missed?

**Comment by danieldavies99** [comment]
@cschol that all makes sense, happy to do that. Quick question, would I break anything by renaming the SequelSequencers repo on GitHub?

**Comment by cschol** [comment]
I would wait until after the slug renaming. I just have to update the submodule name at that time.

**Comment by danieldavies99** [comment]
@cschol Here is a commit on the sequel sequencer repo with the new modules: [67199f7ca79d36de9e70a65ffd3dee393adbbb54](https://github.com/danieldavies99/sequel/commit/67199f7ca79d36de9e70a65ffd3dee393adbbb54)

I haven't added the plugin slug change the plugin.json in this commit, is that correct? or should I change the slug now?

*Edit* I've re-read your above replies and I understand what you're saying now, please could you re-open this issue and use the commit from this message, you can also close/delete the other Issue I created (https://github.com/VCVRack/library/issues/715)

I'll reopen this issue after 2.1.1 is released with the slug change.

Also looking at GitHub docs, if I rename my repo, all links should still redirect, but to avoid confusion Id probably change the submodule URL anyway

**Comment by danieldavies99** [comment]
Oops I screwed up and somehow the slug did change in the above commit, I will submit a new commit shortly

**Comment by danieldavies99** [comment]
New commit: [55c50dcfd5a4e426dc42efaa6afb8f754f51c7a2](https://github.com/danieldavies99/sequel/commit/55c50dcfd5a4e426dc42efaa6afb8f754f51c7a2)

**Comment by danieldavies99** [comment]
@cschol  hi, I've been working on an update that will hopefully be ready for release fairly soon, I've checked and it looks like you've added the [slug change I requested,](https://github.com/VCVRack/Rack/commit/dda07a6ae08eae9944857e6c9ccaffb0552ae3a4) I assume now I just need to update the slug in plugin.json ? 

Also, what's the process for me renaming my GitHub repo? based on what I've read on the GitHub docs I think that all the old links should work after a rename but I don't want to accidentally break anything.

Cheers!

**Comment by cschol** [comment]
Yes. You can submit the updated `plugin.json`. Then you can rename the repo and I will update the submodule to point to the new name. Like you said, the old name should work, but I want to update the name in the submodule anyway.

**Comment by danieldavies99** [comment]
@cschol @AndrewBelt Hi both, please see the latest version commit v2.2.2: ~~https://github.com/danieldavies99/sequel/commit/16144ead5417d8d0b91a8f40481651895fa1b342~~ (see later comment for up to date commit)

I have added 3 new modules, 2 of which are hidden in the plugin manifest for the time being as they are WIP. One new module (sequel save) will be added as part of this latest update.

I have also updated the plugin slug to "DanielDavies" as per our previous conversation. Also just to confirm, I'm okay to rename the repo to reflect the new plugin name as well? just give me the word and I will do it

Also pgatt has built for ARM64 and all appears to be working (I haven't included builds in my repo though), please let me know if there's anything else I need to do in that regard.

Cheers!

*edit* new commit link, 16/08/2023 - added exit crash fix

**Comment by cschol** [comment]
Go ahead now and rename your repo and I will update the URL in the library.

**Comment by danieldavies99** [comment]
@cschol I have now renamed the repo https://github.com/danieldavies99/DanielDaviesVCV

Also, a crash-on-exit bug was brought to my attention last night, we thought we'd fixed it yesterday but actually more changes were needed which I've just now merged. 

Please can you use this update commit instead: https://github.com/danieldavies99/DanielDaviesVCV/commit/046e8a736fe2768d34cbacc06c9a1630ca621cca

Cheers!

**Comment by AndrewBelt** [comment]
Can you change this slug back to `SequelSequencers`? It's already on the VCV Library https://library.vcvrack.com/SequelSequencers and slug changes are a nightmare with no real benefit. Users will only see "Daniel Davies" since it's the plugin name and brand.

**Comment by danieldavies99** [comment]
ok no problem, new commit: https://github.com/danieldavies99/DanielDaviesVCV/commit/44d2f80efc93c071d1195ab67ede88eafa03974c although I just want to confirm this is definitely correct because I think you already laid some foundations for this a while ago: https://github.com/VCVRack/Rack/commit/dda07a6ae08eae9944857e6c9ccaffb0552ae3a4

It doesn't majorly bother me either way

**Comment by danieldavies99** [comment]
@cschol @AndrewBelt Hello again, please see this commit https://github.com/danieldavies99/DanielDaviesVCV/commit/aca0a4d2497f934b65cac766079b63a8b055ca79 for the latest version of the DanielDaviesVCV modules (v2.3.0) I have added two new modules: Bend and James.

Please re-open this thread at your convenience.

Cheers!

*edit* new commit hash

**Comment by danieldavies99** [comment]
Hi both, Apologies for the rapid releases, but I've got one more. New commit hash: https://github.com/danieldavies99/DanielDaviesVCV/commit/75587c551c8eb27910b343c34e1fa16cee0b08f6

fixes some aliasing issues I was facing with the Bend oscillator module.

Cheers!

**Comment by danieldavies99** [comment]
Hi @cschol @AndrewBelt I have a new release I would like to submit, commit: https://github.com/danieldavies99/DanielDaviesVCV/commit/143dbb35862dc973f1038090368c7fc2fb82804b

New Module: Knot
Dark Panels
Add ability to change cable colour scheme via right click menu on blank 3 and blank 5 modules.
Major file structure refactor

Cheers!

**Comment by danieldavies99** [comment]
Hi both, if it's not too much hassle, could we use this commit now instead? https://github.com/danieldavies99/DanielDaviesVCV/commit/70f443ae84bfa1751f8af804682b7b73a6d851df

v2.4.1:
Optimizations to my Quantify module
